yarn.lock

```diff
@@ -482,6 +482,13 @@
   resolved "https://registry.yarnpkg.com/@sindresorhus/is/-/is-0.14.0.tgz#9fb3a3cf3132328151f353de4632e01e52102bea"
   integrity sha512-9NET910DNaIPngYnLLPeg+Ogzqsi9uM4mSboU5y6p8S5DzMTVEsJZrawi+BoDNUVBa2DhJqQYUFvMDfgU062LQ==
 
+"@solana/spl-token-registry@^0.2.3178":
+  version "0.2.3178"
+  resolved "https://registry.yarnpkg.com/@solana/spl-token-registry/-/spl-token-registry-0.2.3178.tgz#246415cfc92052879ff52630b3662e3787bba090"
+  integrity sha512-R5h2tmUWUxRrSQOuTKD0UakRH7R1246MMSq6Hls7XdjWeoY+t3lHieODiNKLmEiFJjhTgP9275qzCe8tv9F8ng==
+  dependencies:
```

Comment:
{
  "chainId": 101,
  "address": "Gqiou284yNmVtTGVVM8guof4DaQtbrEDC4kgjNHZbbo4",
  "symbol": "SHIVX",
  "name": "ShivX Privacy",
  "decimals": 4,
  "logoURI": "https://github.com/ShivX10/Logo/blob/main/Edit-ShivX-Logo_page-0001-e1646994033934-102x105.jpg",
  "extensions": {
    "serumV3Usdc": "EFugR6DusHXLfzbbMEQFCmLq5n4k5VTWGcv1MC5yS4BT",
    "twitter": "https://twitter.com/ShivXprivacy",
    "website": "https://www.ShivX.in/"
  }
}